bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/data/repository/NetworkRepository.kt

```diff
@@ -0,0 +1,15 @@
+    package network.bisq.mobile.domain.data.repository
+    
+    import kotlinx.coroutines.delay
+    
+    object NetworkRepository {
```

Comment:
just a heads up this one is still pending @nostrbuddha - BisqStats is availa now so it should be a SingleObjectRepository implementation

Reply:
Yes. will do this.

Reply:
@nostrbuddha sorry I've analysed the case further and it doesn't make sense because that code is actually a service code, I'm moving it now to the presenter for now with a TODO hint for the time I do the networking stuff for the `xClients`

if we do it that way we won't need to extend the SplashPresenter in the `androidNode`, just change the networking service leaving  the presentation intact (as it should because it's not really a UI change :D )